include/Graphs/CFLGraph.h

```diff
@@ -36,6 +36,7 @@
 #include <regex>
 #include "CFL/CFLGrammar.h"
 #include "Graphs/GenericGraph.h"
+#include "Graphs/ConsG.h"
```

Comment:
We should not include ConsG.h because this is a general CFLGraph. I would suggest creating a CFLGraphBuilder.h/cpp class to build a CFLGraph. Move specific classes and data structures into the `CFLGraphBuilder` class

Reply:
Put CFLGraphBuilder.h/cpp under the CFL folder.

Reply:
Yes, I will work on CFLGraphBuilder